src/components/JobConfiguration.vue

```diff
@@ -324,7 +327,14 @@ export default defineComponent({
     },
     updateRunTime(ev: CustomEvent, job: any) {
       if (job) {
-        job.runTime = handleDateTimeInput(ev['detail'].value)
+        const currTime = DateTime.now().toMillis();
```

Comment:
I think we should handle the same case for initial job configuration component.

Reply:
Sure, sir. Shall I go ahead and make the same changes there?

Reply:
Yes